.github/workflows/publish-firefox.yml

```diff
@@ -104,39 +99,8 @@ jobs:
       - name: Set Firefox for Android compatibility
         if: steps.sign_step.outputs.status == '0'
         run: |
-          node <<'EOF'
-          const jwt = require('jsonwebtoken');
-          const { execSync } = require('child_process');
-          const issuedAt = Math.floor(Date.now() / 1000);
-          const payload = {
-            iss: process.env.AMO_JWT_ISSUER,
-            jti: Math.random().toString(),
-            iat: issuedAt,
-            exp: issuedAt + 60,
-          };
-          const secret = process.env.AMO_JWT_SECRET;
-          const token = jwt.sign(payload, secret, { algorithm: 'HS256' });
-
-          const version = process.env.VERSION;
-          const slug = process.env.EXTENSION_SLUG;
-          const data = JSON.stringify({
-            compatibility: {
-              firefox: { min: "109.0", max: "*" },
-              android: { min: "120.0", max: "*" }
-            }
-          });
-
-          const curlCmd = [
-            'curl', '-X', 'PATCH',
-            `"https://addons.mozilla.org/api/v5/addons/addon/${slug}/versions/${version}/"`,
-            '-H', `"Authorization: JWT ${token}"`,
-            '-H', '"Content-Type: application/json"',
-            '--data', `'${data}'`
-          ].join(' ');
-
-          console.log(`[INFO] Running: ${curlCmd}`);
-          execSync(curlCmd, { stdio: 'inherit' });
-          EOF
+          npm install jsonwebtoken node-fetch
```

Comment:
node 18 does not have native fetch? also maybe its worth to pin versions in packages